packages/ava/src/advisor/advise-pipeline/data-to-advices.ts

```diff
@@ -203,14 +208,14 @@ function applySmartColor(
  * @returns chart list [ { type: chartTypes, spec: antv-spec, score: >0 }, ... ]
  */
 export function dataToAdvices(
-  data: DataRows,
+  data: Datum[],
```

Comment:
```suggestion
  data: Data,
```
`import type { Specification, Data } from '../../common/types';`

Reply:
done